config/rbac/role_binding.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
   name: manager-rolebinding
+  namespace: system
```

Comment:
Should this be `eraser-system`?

Reply:
by default the roles are in "system" and during deployment it changes to "eraser-system" since the namespace is configurable

Reply:
kubebuilder automatically prefixes these